haas/model.py

```diff
@@ -243,14 +242,14 @@ class Network(Model):
     # The project to which the network belongs, or None if the network was
     # created by the administrator.  This field determines who can delete a
     # network.
-    creator_id = Column(String,ForeignKey('project.id'))
+    creator_id = Column(Integer,ForeignKey('project.id'))
```

Comment:
Works for me. Since @apoorvemohan is unlikely to get to this until he gets back, I'm going to open up a new pr from my branch with this extra change.